common/views/components/AppContext/AppContext.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import { createContext, useState, useEffect, type Node } from 'react';
+// $FlowFixMe
```

Comment:
Why?

Reply:
I'd copied over how we're doing `createContext` elsewhere, but I think you're right to question this since we know the shape of the type annotation for `T`. Will update.

```
<T>(
  defaultValue: T,
  calculateChangedBits: ?(a: T, b: T) => number
) => React$Context<T>
(alias) createContext<{}>(defaultValue: {}, calculateChangedBits?: (prev: {}, next: {}) => number): Context<{}>
import createContext
Missing type annotation for `T`. `T` is a type parameter declared in  function type [1] and was implicitly instantiated at  call of `createContext` [2].Flow(InferError)
```